modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -313,6 +314,17 @@ public void testDeleteObjectRelationship() {
 						objectRelationship.getDeletionType(), true,
 						objectRelationship.getLabelMap()));
 			});
+
+		ObjectRelationship systemObjectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition2.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				StringUtil.randomId(), true,
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
```

Comment:
The variable should be declared close to its use. 
Can you move this code to before the `AssertUtils.assertFailure(ObjectRelationshipSystemException.class, ...) `?